src/unstructured_client/_hooks/custom/split_pdf_hook.py

```diff
@@ -332,67 +345,66 @@ def _decode_content_disposition(self, content_disposition: bytes) -> dict[str, s
         parameters_dict = {p[0]: p[1].strip('"') for p in parameters}
         return parameters_dict
 
+    @staticmethod
```

Comment:
TBH I'm not a fan of static methods, so I if you want me to make all of them a regular functions, feel free to vote here.

Converting this method to static was necessary to pickle arguments correctly (required by ProcessPoolExecutor). 
![image](https://github.com/Unstructured-IO/unstructured-python-client/assets/64484917/d8169001-be23-46fc-85c6-8e214faa26c5)

Reply:
this is the part that really sets process and thread apart:
- generally a regular function is better as this reinforces the design that the code runs in multiprocessing does not require context more than whats in the function and variables passed in
- multithreading shares the context in the process they spawn from

Reply:
done, refactored